virtual_ship/planning_error.py

```diff
@@ -0,0 +1,7 @@
+"""PlanningError Exception."""
+
+
+class PlanningError(RuntimeError):
+    """An error when checking the schedule or during sailing."""
```

Comment:
Can this be any more specific? Provide some further info when/where/why the error was caused. Otherwise, we'll have to do a lot of debugging for students...